src/spikeinterface/extractors/neoextractors/plexon.py

```diff
@@ -60,10 +60,9 @@ class PlexonSortingExtractor(NeoBaseSortingExtractor):
     def __init__(self, file_path):
         neo_kwargs = self.map_to_neo_kwargs(file_path)
         self.neo_reader = NeoBaseSortingExtractor.get_neo_io_reader(self.NeoRawIOClass, **neo_kwargs)
-        self.neo_reader.parse_header()
```

Comment:
This is already pasred in the method above (62) that gets the neo reader.

Reply:
thanks for the catch. Very strange why we have this here.
I think this is me sorry.